cognite/client/data_classes/simulators/routine_revisions.py

```diff
@@ -666,6 +691,118 @@ def as_write(self) -> SimulatorRoutineRevisionWriteList:
         )
 
 
+class SimulatorRoutineStageList(UserList[SimulatorRoutineStage]):
+    """List of simulator routine stages with pandas conversion capabilities."""
+
+    def __init__(self, initlist: Sequence[SimulatorRoutineStage] | None = None) -> None:
+        super().__init__(list(initlist) if initlist is not None else None)
```

Comment:
Why keep the `None` option? Shouldn't that be "empty sequence" instead?

Reply:
Replaced `None` with empty list